common/src/models/project.rs

```diff
@@ -57,13 +57,22 @@ pub struct AdminResponse {
     pub account_name: String,
 }
 
-pub fn get_table(projects: &Vec<Response>) -> String {
+pub fn get_table(projects: &Vec<Response>, filter: Option<State>) -> String {
     if projects.is_empty() {
         format!(
             "{}\n",
             "No projects are linked to this account".yellow().bold()
         )
     } else {
+        let projects = match filter {
```

Comment:
So currently all projects are fetched, then filtered. We should rather apply the filter when fetching the projects from the database, and then only send the filtered projects to the client. We could add a route to the gateway API for this, with the desired state passed in as a path parameter, so something like "/projects/:state". And then make a handler that selects from the db the projects owned by this user with the desired state.

Reply:
I've pushed a new commit to try and amend this.